e2e-tests/insurance/cypress/e2e/journeys/policy/loss-payee-details/loss-payee-details.spec.js

```diff
@@ -115,12 +118,46 @@ context('Insurance - Policy - Loss payee details page - As an exporter, I want t
   });
 
   describe('form submission', () => {
-    it(`should redirect to ${CHECK_YOUR_ANSWERS}`, () => {
-      cy.navigateToUrl(url);
+    describe(IS_LOCATED_IN_UK, () => {
+      beforeEach(() => {
+        cy.navigateToUrl(url);
+      });
+
+      it(`should redirect to ${LOSS_PAYEE_FINANCIAL_UK_ROOT}`, () => {
+        cy.completeAndSubmitLossPayeeDetailsForm({});
+
+        cy.assertUrl(lossPayeeFinancialUKUrl);
+      });
+
+      describe('when going back to the page', () => {
+        it('should have the submitted values', () => {
+          cy.checkValue(fieldSelector(NAME), POLICY[NAME]);
+
+          const radioFieldId = `location-${IS_LOCATED_IN_UK}`;
+          cy.assertRadioOptionIsChecked(fieldSelector(radioFieldId).input());
+        });
+      });
+    });
+
+    describe(IS_LOCATED_INTERNATIONALLY, () => {
+      beforeEach(() => {
+        cy.navigateToUrl(url);
+      });
+
+      it(`should redirect to ${LOSS_PAYEE_FINANCIAL_UK_ROOT}`, () => {
+        cy.completeAndSubmitLossPayeeDetailsForm({ locatedInUK: false });
+
+        cy.assertUrl(lossPayeeFinancialUKUrl);
+      });
 
-      cy.completeAndSubmitLossPayeeDetailsForm({});
+      describe('when going back to the page', () => {
+        it('should have the submitted values', () => {
+          cy.checkValue(fieldSelector(NAME), POLICY[NAME]);
 
-      cy.assertUrl(checkYourAnswersUrl);
+          const radioFieldId = `location-${IS_LOCATED_INTERNATIONALLY}`;
+          cy.assertRadioOptionIsChecked(fieldSelector(radioFieldId).input());
+        });
+      });
```

Comment:
I think this could be half the size

```js
describe('form submission', () => {
  beforeEach(() => {
    cy.completeAndSubmitLossPayeeDetailsForm({});
  });

  describe('when going back to the page', () => {
    it('should have the submitted values', () => {
      cy.checkValue(fieldSelector(NAME), POLICY[NAME]);

      // assertions
    });
  });
});
```